model_server/pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[build-system]
+requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "stampy_nlp_model"
+dynamic = ["version"]
+authors = [
+    {name = "ccstan99", email = "hello@stampy.ai"},
+    {name = "Daniel O'Connell", email = "github@ahiru.pl"},
+]
+description = "A simple server to encapsulate LLMs as microservices"
+readme = "README.md"
+requires-python = ">=3.8"
+license = {text = "MIT"}
+dependencies = [
+    "starlette",
+    "asyncio",
+    "uvicorn",
+    "sentence-transformers",
+    "torch",
+    "transformers",
+]
```

Comment:
I considered splitting this even further, to have different dependencies, but ultimately decided to keep it simple. It's possible to do magic with the `MODEL_TYPE` setting so it autoinstalls the appropriate dependencies. The actual library only weighs around 1MB, so that shouldn't be a concern

Reply:
OK, sounds good.